File: C/section07/S07E06.c
#include <stdio.h>

int main() { 
    int number;

    printf("Please enter the number you wish to calculate from 1 to 10.\n");
    scanf("%d", &number);

    while(number > 10 || number < 1) {
        printf("The number must be between 1 and 10. \n");
        printf("Please enter the number you wish to calculate from 1 to 10.\n");
    scanf("%d", &number);
    }

    for (int count = 1; count <= 10; count++) {
        printf("\n%d X %d = %d", number, count, number * count);
    }

    return 0;    
}

File: Python/section07/S07e06.py
number = 0

number = int(input("Please enter a number between 1 and 10 to calculate it's times table: "))
while number < 1 or number > 10:
    number = int(input("The number must be between 1 and 10, try again: "))
print("\n{0} Times table".format(number))
for index in range(1, 11):
    print("{0} X {1} = {2}".format(number, index, number * index))